### PR TITLE
Clarify perfectly synchronized

### PR DESCRIPTION
**Pull request by kste**
Add explanation on what is meant by perfectly synchronized.

### CODE REVIEW

**Review by crypto4a — CHANGES_REQUESTED** [review]
One small nitpick: please change "during procedure which load" to "during procedures which load". Other than that it looks good to me.

File: draft-wiggers-hbs-state.md [hunk 1]
```diff
@@ -390,7 +390,8 @@ main concerns here are
 A system may have a version of the private key stored in non-volatile memory
 (e.g. a disk) and will load it into volatile memory (e.g. RAM) while
 processing. Here, an implementer MUST ensure that these are always perfectly
-synchronized [MCGREW]. This can be particularly challenging if there are
+synchronized [MCGREW], meaning that no parts of the system are allowed to read
+any version of the key during procedure which load, write or modify keys. This can be particularly challenging if there are
```
Comment (BashiriK) [inline draft-wiggers-hbs-state.md:394]: As a non-native speaker: du we need "procedures" or "procedure"

Comment (thomwiggers) [reply draft-wiggers-hbs-state.md:394]: This should be plural AFAIK

Comment (thomwiggers) [reply draft-wiggers-hbs-state.md:394]: as also pointed out by @crypto4a's comment

**Comment by thomwiggers** [comment]
I've pluralized it

File: draft-wiggers-hbs-state.md (hunk 1 above)
Comment (BashiriK) [reply draft-wiggers-hbs-state.md]: Oh, now I see! Feels good to have, independently, the same thoughts as Jim.

**Review by BashiriK — APPROVED** [review]

**Pull request merged**